contexts/DelegationContext.tsx

```diff
@@ -40,8 +40,22 @@ export interface DelegationContextState {
   ignoreReceivedRequestToBeDelegate: (delegatorAddress: string) => void;
   terminateRelationshipWithDelegate: () => void;
   terminateRelationshipWithDelegator: () => void;
-  getDelegationDataLoading: () => boolean;
-  getDelegationDataFetching: () => boolean;
+  isLoading: boolean;
```

Comment:
the loading situation for the delegation context is a bit complicated because it involves a lot of interconnected values. Since I can't see any situations where we could benefit from treating the various ...isLoading items separately, I've added convenient values that aggregate them: isLoading, isMutating, isBusy (either loading or mutating).

I'm still passing along all of these loading states so that they could be used if need be, but I doubt we'll want to.

Reply:
makes sense, though hopefully we will be movign away from these generalized data hooks in the future